### PR TITLE
Conversation API WhatsApp Updates

### PR DESCRIPTION
**Pull request by nickjn92**

### CODE REVIEW

File: docs/conversation/conversation-channel-support/conversation-whatsapp.md [hunk 1]
```diff
@@ -259,7 +259,13 @@ Conversation API POST `messages:send`
 
 **Interactive Template**
 
-WhatsApp allows sending interactive templates with up to three buttons.
+WhatsApp offers two types of predefined buttons
+* *Quick Reply* - Presents a button with a predefined text which then returns a callback with a defined payload when clicked
+* *Call-to-Action* - Presents a clickable action in WhatsApp to either call a phone number or to visit a website   
+
+These buttons can be attached to text or media templates. However you are only allowed to register templates that has at most of the types, i.e. you can not have a registered template that has both call-to-action and quick reply buttons.
```
Comment (Krattan) [inline docs/conversation/conversation-channel-support/conversation-whatsapp.md:266]: you are only allowed to register templates that has at most _one_ of the types,

File: docs/conversation/conversation-channel-support/conversation-whatsapp.md [hunk 2]
```diff
@@ -259,7 +259,13 @@ Conversation API POST `messages:send`
...
+
+When using quick reply buttons in your templates you can have at most 3 buttons and while using call-to-action buttons you can have at most 2 buttons.
```
Comment (Krattan) [inline docs/conversation/conversation-channel-support/conversation-whatsapp.md:268]: consider changing numbers to three and two

File: docs/conversation/conversation-channel-support/conversation-whatsapp.md [hunk 3]
```diff
@@ -276,10 +282,37 @@ The quick_reply payload field must be set even when empty. For example for quick
     "button[0]quick_reply[1]payload": ""
  ```
 
-Below is an example of sending a template with a quick reply button at index 0 and two URL buttons at index 1 and 2 respectively. The template have also one header and one body
-parameters.
+Below is an example of sending a template with three quick reply buttons The template also have one header and one body parameter.
```
Comment (matokeni) [inline docs/conversation/conversation-channel-support/conversation-whatsapp.md:285]: Missing a dot between the sentences, and "The template also has"

Comment (nickjn92) [reply docs/conversation/conversation-channel-support/conversation-whatsapp.md]: Changed

File: docs/conversation/conversation-channel-support/conversation-whatsapp.md (hunk 2 above)
Comment (nickjn92) [reply docs/conversation/conversation-channel-support/conversation-whatsapp.md]: Changed

File: docs/conversation/conversation-channel-support/conversation-whatsapp.md (hunk 1 above)
Comment (nickjn92) [reply docs/conversation/conversation-channel-support/conversation-whatsapp.md]: Changed

**Review by Krattan — APPROVED** [review]

**Pull request merged**